Dockerfile

```diff
@@ -0,0 +1,36 @@
+FROM debian:8-slim AS pim
+MAINTAINER pierre.allard@akeneo.com
+WORKDIR /home/akeneo/pim-docs/
+
+# Sphinx installation
+RUN apt-get update && \
+    apt-get install -y python-pip git && \
+    apt-get clean && \
+    rm -rf /var/lib/apt/lists/*
+RUN pip install --upgrade pip
+RUN pip install sphinx~=1.5.3
+RUN pip install git+https://github.com/fabpot/sphinx-php.git && \
+    pip install git+https://github.com/mickaelandrieu/sphinxcontrib.youtube.git
+
+# Akeneo PIM installation
+ENV DEBIAN_FRONTEND=noninteractive
+RUN apt-get update && \
+    apt-get install -y wget unzip && \
+    apt-get install -y php5 mysql-server apache2 libapache2-mod-php5 php5-cli php5-apcu php5-mcrypt php5-intl php5-mysql php5-curl php5-gd mongodb php5-mongo && \
+    apt-get clean && \
```

Comment:
You can do better to clean and minimize the image size:
https://github.com/akeneo/Dockerfiles/blob/master/php/Dockerfile#L11